build.gradle

```diff
@@ -64,8 +64,7 @@ dependencies {
   testImplementation gradleTestKit()
   testImplementation "junit:junit:$junitVersion"
   testImplementation "org.assertj:assertj-core:$assertJVersion"
-  testImplementation "org.powermock:powermock-module-junit4:$powermockVersion"
-  testImplementation "org.powermock:powermock-api-mockito2:$powermockVersion"
+  testImplementation "org.mockito:mockito-core:${mockitoVersion}"
```

Comment:
Switched from powermock to mockito due to powermock/jdk 11 issues e.g. the test `NexusIqScanTaskTest.testScan_realWithScanTargets` would print warnings for illegal reflective access. The approach taken with mockito does not.